src/DeconvUtils.jl

```diff
@@ -61,7 +61,7 @@ function solveFactorMeasurements(dfg::AbstractDFG,
         if !r.g_converged
           nsm = getSample(fcttype, 1)
           for count in 1:length(meas)
```

Comment:
wow, okay good catch.  Now I'm wondering if this `length(meas)` is right?  `meas::Tuple`, but element `idx` is usually element of `size(meas[1],2)`

Reply:
ah wait that is right

Reply:
already have `idx = 1:N` as outer loop

Reply:
this is all right

Reply:
actually wait